packages/core/src/common/message-service.ts

```diff
@@ -73,6 +152,43 @@ export class MessageService {
         return this.client.showMessage({ type, text });
     }
 
+    /**
+     * Shows the given message as a progress.
+     *
+     * @param message the message to show for the progress. If not specified otherwise the message will be cancelable.
```

Comment:
As message is a `ProgressMessage` I wanted to underline the fact that if not specified otherwise via the `ProgressMessageOptions` the message will be one which can be canceled. However I could also remove the information here as it's documented for `ProgressMessageOptions` too.

Reply:
I was fine with the comment, just the phrasing seemed off:

```
If not specified, the message will be canceable.
```

or 

```
Canceable if no message is specified.
```